docs/table_design/Data_distribution.md

```diff
@@ -1,107 +1,209 @@
 # Data distribution
 
-When you create a table, you can specify the data distribution method by configuring the partitioning and bucketing strategy in the table. An appropriate data distribution method helps evenly distribute data across the nodes of your StarRocks cluster. It can also reduce the data scanned and make full use of the concurrency of the cluster during queries, thereby improving query performance.
+Configuring appropriate partitioning and bucketing at table creation data can help to achieve even data distribution and improve query performance. Even data distribution means dividing the data into subsets according to certain rules and distributing them evenly across different nodes. It can also reduce the data scanned and make full use of the concurrency of the cluster during queries, thereby improving query performance.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/e69b2a2d-d518-4e1c-9125-0cf6fd0bd84a)
It seems "data" is redundant because with it the whole sentence is grammatically unnatural.